Expecto/Expect.fs

```diff
@@ -228,7 +229,77 @@ let containsAll (actual : _ seq)
         %s
         Extra elements in `actual`:
         %s"
-              format (formatSet axs) (formatSet exs) (formatSet missing) (formatSet extra)
+              format (formatResult axs) (formatResult exs) (formatResult missing) (formatResult extra)
+  |> Tests.failtest
+
+let inline private except(baseMap: Map<_,int>, exceptMap: Map<_,int>, isExcept: bool) =
+  let getMapValue (map: Map<_, int>) (element: _) = 
```

Comment:
Prefer not using `element : _`; instead just write `element` if the type annotation is not used.